### PR TITLE
Restaurant Page

### PR DESCRIPTION
**Pull request by mapra99**
In this solo project I built a restaurant homepage using modules, npm and webpack.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```
Comment (jofer86) [inline .eslintrc.json]: - I'm gonna have to ask you to please configure this file as the one [right here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json); make it look like this 
image; lint, if necessary and you are golden!

**Review by jofer86 — CHANGES_REQUESTED** [review]
## :wave: :wave: Hello @mapra99, I'm here to review this PR, let's take a look shall we :eye: :eye: :eye: 

- Everything looks good, your code is properly modulated, and you took care of every requirement asked by this project, however, there is one little thing I'm gonna have to require of you in order to approve this PR, that is to configure your .eslintrc.json file to match the one on the microverse linters repo; tend to that, lint if it is necessary and request for another code review and you shall be good to go!!
--------------------------------------------------------------------
Cheers, 
Jorge Fernando Rincón Figueroa.
Technical Support Engineer.
**Slack: @jofer86**

File: .eslintrc.json (hunk 1 above)
Comment (mapra99) [reply .eslintrc.json]: Thanks @jofer86!  I thought I already did that, I'll look into it

**Review by shloch — APPROVED** [review]
## Great job @mapra99 

accomplished

You can merge

**Pull request merged**